test/e2e/skopeo.go

```diff
@@ -24,7 +24,19 @@ const (
 )
 
 func openshiftRegistryAuth(client operatorclient.ClientInterface, namespace string) (string, error) {
-	sa, err := client.KubernetesInterface().CoreV1().ServiceAccounts(namespace).Get(context.TODO(), BuilderServiceAccount, metav1.GetOptions{})
+
+	var sa *corev1.ServiceAccount
+	var err error
+
+	// wait for the builder service account to exist and contain image pull secrets
+	err = waitFor(func() (bool, error) {
```

Comment:
Does it make it more clear to wrap this in a ginkgo `Eventually` instead?

Reply:
since this is an utility method I thought it better to use waitFor and let the error  be checked up the call stack. wdyt?

Reply:
Makes sense!